packages/http-server-js/src/scripts/scaffold/data-mocks.ts

```diff
@@ -186,34 +190,35 @@ function mockUnion(ctx: JsContext, module: Module, union: Union): string | undef
  * @returns A JavaScript string representation of a suitable mock value for the scalar type
  */
 function mockScalar(ctx: JsContext, module: Module, scalar: Scalar): string | undefined {
-  if ($.scalar.isBoolean(scalar) || $.scalar.extendsBoolean(scalar)) {
+  const tk = $(ctx.program);
```

Comment:
do you find this better than just inlining `$(program)`

Reply:
My main motivation was to remove the overhead of calling `$(program)` more than once - particularly in this function where it could be called many times.